src/include/function/string/functions/ends_with_function.h

```diff
@@ -1,17 +1,27 @@
 #pragma once
 
 #include "common/types/ku_string.h"
-#include "function/string/functions/find_function.h"
 
 namespace kuzu {
 namespace function {
 
 struct EndsWith {
     static inline void operation(common::ku_string_t& left, common::ku_string_t& right,
         uint8_t& result) {
-        int64_t pos = 0;
-        Find::operation(left, right, pos);
-        result = (pos == left.len - right.len + 1);
+        if (right.len > left.len) {
```

Comment:
Why can't we reuse the find function? 
If there is a bug in the find function we should fix that
And this new code doesn't handle utf-8 correctly.

Reply:
Find is logically incorrect; find("bbbb", "b") gives index 1. We then conclude that "b" is not a suffix of "bbbb"